cmd/validator/core/command.go

```diff
@@ -57,6 +59,19 @@ func run(config Params, addr string, threads int, processorNum int, printCurl bo
 		ErrorLog.Fatalf("indexer-url parameter is required.")
 	}
 
+	if errorLogFile != "" {
+		_, err := os.Stat(errorLogFile)
+		if !os.IsNotExist(err) {
+			ErrorLog.Fatalf("Error log already exists: %s", errorLogFile)
```

Comment:
so on a restart it will fail or am i misunderstanding this?

Reply:
also double negative in !os.IsNotExist

Reply:
Yes, I didn't think it makes any sense to restart. If you ran a long test you risk deleting results on accident. If you appending to the end then the second test results are buried deep inside a large file.

Open to suggestions if this behavior isn't desired.

For the double negative, the go file interfaces are kinda weird and the double negative is the only sure way I know of to verify that the correct condition is being checked.

https://stackoverflow.com/a/12518877/204023